prometheus_client/gc_collector.py

```diff
@@ -12,7 +13,7 @@ class GCCollector(object):
     """Collector for Garbage collection statistics."""
 
     def __init__(self, registry=REGISTRY):
-        if not hasattr(gc, 'get_stats'):
+        if not hasattr(gc, 'get_stats') or platform.python_implementation() == 'PyPy':
```

Comment:
Wouldn't it be better to limit this to CPython?

Reply:
You right.